.github/workflows/phpunit.yml

```
@@ -47,7 +48,8 @@ jobs:
           --health-retries 5
     steps:
       - name: Run PHPUnit Action
-        uses: discoverygarden/phpunit-action@v1
+        #uses: discoverygarden/phpunit-action@v1
+        uses: discoverygarden/phpunit-action@fix/auth
```

Should go back to the whatever release version prior to merge:

```suggestion
        uses: discoverygarden/phpunit-action@v1
```